internal/storage/gcs/bucket.go

```diff
@@ -20,6 +20,15 @@ import (
 	"golang.org/x/net/context"
 )
 
+// BucketTypes represents different types of buckets.
+type BucketTypes int
```

Comment:
please rename to BucketTypes as  at one time it can hold only one value, so should not be plural